torch/onnx/utils.py

```diff
@@ -1140,31 +1142,32 @@ def _export(
                 torch.onnx.log(
                     "Exported graph: ", _assign_onnx_node_name(graph, node_names)
                 )
-            if export_type == torch.onnx.ExportTypes.PROTOBUF_FILE:
+            if export_type == _exporter_states.ExportTypes.PROTOBUF_FILE:
                 assert len(export_map) == 0
                 with torch.serialization._open_file_like(f, "wb") as opened_file:
                     opened_file.write(proto)
             elif export_type in [
-                torch.onnx.ExportTypes.ZIP_ARCHIVE,
-                torch.onnx.ExportTypes.COMPRESSED_ZIP_ARCHIVE,
+                _exporter_states.ExportTypes.ZIP_ARCHIVE,
+                _exporter_states.ExportTypes.COMPRESSED_ZIP_ARCHIVE,
             ]:
                 compression = (
                     zipfile.ZIP_DEFLATED
-                    if export_type == torch.onnx.ExportTypes.COMPRESSED_ZIP_ARCHIVE
+                    if export_type
+                    == _exporter_states.ExportTypes.COMPRESSED_ZIP_ARCHIVE
```

Comment:
weird format .. is this normal?

Reply:
If could be black thinking line is too long. lintrunner seems to be happy about it.